app/assets/javascripts/application/translation_workbench.js.coffee.erb

```diff
@@ -131,7 +131,7 @@ class TranslationItem
 
   # @private
   build: ->
-    severity_texts = ['Minor', 'Neutral', 'Major', 'Critical']
+    severity_texts = <%= TranslationChange.reason_severities.map {|s| s[0].titlecase} %>
```

Comment:
I mean the display string should not be `converted` from enum. (I know Shuttle does it several times in other places).
The code can use enum or strings to differentiate the severities. But the display strings need to be loaded from en.yml.

Anyway, I will approve and merge the PR for now.